### PR TITLE
[Refactor] Refactor alert module.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Refactor alert module.

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/AlertController.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public RestResponse deleteAlertConfig(
 
   @Operation(summary = "Send alert config")
   @PostMapping("/send")
-  public RestResponse sendAlert(Long id) throws AlertException {
+  public RestResponse sendAlert(Integer id) throws AlertException {
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/AlertController.java]: I have a minor confusion, is there any problem with using `Long` here ?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkHttpWatcher.java [hunk 2]
```diff
@@ -780,7 +781,7 @@ interface Callback<T, R> {
   /**
    * The situation of abnormal operation alarm is as follows: When the job running mode is yarn per
    * job or yarn application, when the job is abnormal, an alarm will be triggered directly; The job
-   * running mode is yarn session or reome: a. If the flink cluster is not configured with an alarm
+   * running mode is yarn session or remote. If the flink cluster is not configured with an alarm
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkHttpWatcher.java]: From the original text, it should be annotated according to subheadings such as `a`, `b`, and `c`

File: streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/alert/AlertServiceTest.java [hunk 3]
```diff
@@ -193,7 +190,7 @@ void testAlert() {
           String.format("StreamPark Alert: %s %s", application.getJobName(), appState.name());
       sendEmail(subject, html, "****@domain.com");
     } catch (Exception e) {
-      e.printStackTrace();
+      log.error("Failed to send email alert", e);
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/alert/AlertServiceTest.java:193]: 👍

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/AlertType.java [hunk 4]
```diff
@@ -27,27 +35,32 @@
...
-  WE_COM(4),
+  WE_COM(4, WeComAlertNotifyServiceImpl.class),
 
   /** Http callback */
-  HTTP_CALLBACK(8),
+  HTTP_CALLBACK(8, HttpCallbackAlertNotifyServiceImpl.class),
 
   /** Lark */
-  LARK(16);
+  LARK(16, LarkAlertNotifyServiceImpl.class);
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/enums/AlertType.java:54]: 👍

**Review by RocMarshal — COMMENTED** [review]
Hi, @GOODBOY008 Thanks for driving it.
LGTM on the whole. I left a few of comments.
PTAL in your free time. 
Thanks~

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**